### PR TITLE
DevKit updates for 1.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

**Comment by greg0ire** [comment]
:thinking: did I mess up the configuration?

**Comment by greg0ire** [comment]
Why yes, yes I very much did. Sorry. https://github.com/sonata-project/dev-kit/pull/441

File: .travis/install_test.sh [hunk 1]
```diff
@@ -4,7 +4,12 @@ set -ev
 mkdir --parents "${HOME}/bin"
 
 # PHPUnit install
-wget "https://phar.phpunit.de/phpunit-7.phar" --output-document="${HOME}/bin/phpunit"
```
Comment (OskarStark) [inline .travis/install_test.sh:7]: @greg0ire this one is now rewritten.
Do you find some time to make phpunit version configurable on dev-kit?

Comment (OskarStark) [reply .travis/install_test.sh:7]: ping :)

Comment (greg0ire) [reply .travis/install_test.sh:7]: Woops no I didn't find the time, I remember a solution that involves searching for php 5 in the array though

**Pull request merged**